src/Contracts/Models/Workflow.cs

```diff
@@ -1,21 +1,21 @@
-﻿using Newtonsoft.Json;
+﻿// SPDX-FileCopyrightText: © 2021-2022 MONAI Consortium
+// SPDX-License-Identifier: Apache License 2.0
+
+using MongoDB.Bson.Serialization.Attributes;
+using Newtonsoft.Json;
 
 namespace Monai.Deploy.WorkflowManager.Contracts.Models
 {
     public class Workflow
     {
-        [JsonProperty(PropertyName = "name")]
-        public string Name { get; set; }
-
-        [JsonProperty(PropertyName = "version")]
-        public string Version { get; set; }
+        [JsonIgnore]
+        [BsonId]
+        public string Id { get; set; }
 
-        [JsonProperty(PropertyName = "description")]
-        public string Description { get; set; }
+        public string WorkflowId { get; set; }
 
-        [JsonProperty(PropertyName = "informatics_gateway")]
-        public InformaticsGateway InformaticsGateway { get; set; }
+        public int Revision { get; set; }
 
-        public TaskObject[] Tasks { get; set; }
+        public WorkflowSpec WorkflowSpec { get; set; }
```

Comment:
I see you have `JsonProperty` in other models but not this one?

Reply:
Good spot, added